docs/source/distributed.rst

```diff
@@ -36,6 +36,33 @@ MPI supports cuda only iff the implementation used to build PyTorch supports it.
 | barrier    | ✓   | ✘   | ✓   | ✓   | ✓   | ?   |
 +------------+-----+-----+-----+-----+-----+-----+
 
+.. _distributed-basics:
+
+Basics
+------
+
+The `torch.distributed` package provides PyTorch support and communication primitives
+for synchronous multiprocess parallelism across several computation nodes running on one or more
```

Comment:
That's not true. We have `isend` and `irecv` that can be used to implement asynchronous communication. Also, if by asynchronous paralellism you mean asynchronous training, then this can be achieved using regular `send` and `recv` too (just have the parameter server listening on `recv` without specifying the source, and send updates from the training processes)